include/c3d/base.h

```diff
@@ -1,41 +1,148 @@
+/**
+ * @file base.h
+ * @brief Base citro3d functions
+ */
 #pragma once
 #include "buffers.h"
 #include "maths.h"
 
+/// Default command buffer size
 #define C3D_DEFAULT_CMDBUF_SIZE 0x40000
 
+/**
+ * @brief Data type of indicies for use with \ref C3D_DarawElements.
+ */
 enum
 {
-	C3D_UNSIGNED_BYTE = 0,
-	C3D_UNSIGNED_SHORT = 1,
+	C3D_UNSIGNED_BYTE = 0,  ///< Unsigned 8-bit interger
+	C3D_UNSIGNED_SHORT = 1, ///< Unsigned 16-bit interger
 };
 
+/**
+ * @brief Initalizes citro3d
+ * @param[in] cmdBufSize Desired size of GPU command buffer. \ref C3D_DEFAULT_CMDBUF_SIZE should generally be used, unless there are given memory consumption or scene complexity requirements.
+ * @return true if library was initalized sucessfuly, false if there was an error.
+ */
 bool C3D_Init(size_t cmdBufSize);
+
+/**
+ * @brief De-initalizes citro3d
+ * @sa C3D_Init()
+ */
 void C3D_Fini(void);
 
+/**
+ * @brief Gets command buffer usage
```

Comment:
```suggestion
 * @brief Retrieves the current command buffer usage (0.0 to 1.0)
```

Reply:
The documentation for the return parameter says `Fraction of command buffer used` so I would assume the `(0.0 to 1.0)` would be implied.
Either way, I think that should go with the return parameter, not the function brief, feel free to let me know if you disagree.